src/dbus/server/dbus_thread_object_rcp.cpp

```diff
@@ -2010,6 +2010,23 @@ void DBusThreadObjectRcp::DeactivateEphemeralKeyModeHandler(DBusRequest &aReques
     otError error        = OT_ERROR_NONE;
     auto    threadHelper = mHost.GetThreadHelper();
 
+    if (otBorderAgentIsEphemeralKeyActive(threadHelper->GetInstance()))
+    {
+        error = OT_ERROR_INVALID_STATE;
+        aRequest.ReplyOtResult(error);
+        otbrLogInfo("Failed to deactivate Ephemeral Key due to feature is not enabled");
+        return;
```

Comment:
Please follow the single-return convention.

Reply:
Accepted, thanks~